openedx/features/enterprise_support/enrollments/utils.py

```diff
@@ -0,0 +1,104 @@
+"""
+Utils for use in enrollment codebase such as views.
+"""
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist  # lint-amnesty, pylint: disable=wrong-import-order
+from django.db import transaction
+
+from common.djangoapps.student.models import User
+from openedx.core.djangoapps.course_groups.cohorts import CourseUserGroup
+from openedx.core.djangoapps.enrollments import api
```

Comment:
Small suggestion: turn `api` into `enrollments_api`, so it's easier to understand the context of invocations below.